src/solvers/mp/RCPricer.cpp

```diff
@@ -63,24 +63,22 @@ vector<MyVar*> RotationPricer::pricing(double bound, bool before_fathom){
 	double minDualCost = 0;
 	vector<LiveNurse*> nursesSolved;
 
-	for(vector<LiveNurse*>::iterator it0 = nursesToSolve_.begin(); it0 != nursesToSolve_.end();){
+	for(auto it0 = nursesToSolve_.begin(); it0 != nursesToSolve_.end();){
 
 		// RETRIEVE THE NURSE AND CHECK THAT HE/SHE IS NOT FORBIDDEN
 		LiveNurse* pNurse = *it0;
 		bool nurseForbidden = isNurseForbidden(pNurse->id_);
 
+		// cout << "NURSE # " << pNurse->id_ << "    " << pNurse->name_ << endl;
+
 		// IF THE NURSE IS NOT FORBIDDEN, SOLVE THE SUBPROBLEM
 		if(!nurseForbidden){
 
 			// BUILD OR RE-USE THE SUBPROBLEM
 			SubProblem* subProblem = retriveSubproblem(pNurse);
```

Comment:
Sorry, I can't help noticing... but small typo in function name `retrieveSubproblem` instead